### PR TITLE
Security

### PR DESCRIPTION
**Pull request by kylealexanderhackney**
Draft to check changes

### CODE REVIEW

File: RepairsApi/V2/Authorisation/Groups.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+namespace RepairsApi.V2.Authorisation
+{
+    public static class Groups
+    {
+        public static readonly Dictionary<string, PermissionsModel> SecurityGroups = new Dictionary<string, PermissionsModel>
+        {
+            { "repairs-hub-frontend-staging", (SecurityGroup.AGENT, null) },
+            { "repairs-hub-frontend-staging-contractors-alphatrack", (SecurityGroup.CONTRACTOR, "ASL") },
+            { "repairs-hub-frontend-staging-contractors-purdy", (SecurityGroup.CONTRACTOR, "PCL") },
```
Comment (LBHMKeyworth) [inline RepairsApi/V2/Authorisation/Groups.cs:13]: Is this only hard coded for go-live?

Comment (kylealexanderhackney) [reply RepairsApi/V2/Authorisation/Groups.cs:13]: no this needs to be configure per environment. do we have a to do this in AWS?

**Review by LBHMKeyworth — APPROVED** [review]
Looks good - pending the amendment to allow the groups permissions into param store for now.

**Review by LBHMKeyworth — APPROVED** [review]
Nice!

**Pull request merged**